resource-aggregate/.plantuml

```diff
@@ -296,4 +296,28 @@ ResourceAggregate --> Bus : DeviceMetadataUpdated (Canceled=true)
 ResourceAggregate -> Client : CancelPendingMetadataUpdatesResponse
 deactivate Client
 deactivate ResourceAggregate
-@enduml
\ No newline at end of file
+@enduml
+
+@startuml
+title "Delete Devices"
+
+entity "OCF Server" as Server
+participant "Gateway" as Gateway
+participant "Resource Aggregate" as ResourceAggregate
+participant "Authorization Server" as AuthServer
+
+activate ResourceAggregate
+activate Server
+activate Gateway
+activate AuthServer
+Server -> Gateway : DeleteDevicesRequest
+Gateway-> ResourceAggregate : DeleteDevicesRequest
```

Comment:
Could you please include also EventBus into this diagram and visualize events which these components, RA, AS are sending?

Reply:
Not sure what you mean here. This actually deletes all events for given devices from the RA database and all devices from the Auth database ...?